content/user/teamserver/guide/integrations/integrations.md

```diff
@@ -0,0 +1,73 @@
+<!--
+title: "Integrations"
+description: "Adding Integrations to TeamServer"
+tags: "integrations hipchat generic slack webhook teamserver"
+-->
+
+## Slack Integration
+
+TeamServer now supports Slack integration! With this new connection you can receive notifications from TeamServer to your configured Slack instance in a similar format to in-app notifications.
+
+## Setup
+
+To add, go to your teams 'Build' settings in Slack:
+
+1. Add a new 'Incoming Webhooks' custom integration.
+2. Choose an appropriate channel to send messages to.
+3. Copy the 'Webhook URL'.
+
+To add in TeamServer:
+
+1. Go to the new 'Integrations' tab in Organization Settings.
+2. Click 'Connect' in the Slack row.
+3. Name the integration and paste the URL.
+4. Select the appropriate application(s) to filter on.
+5. Click save.
+
+You are now connected!
+
+## HipChat Integration
+
+TeamServer now supports HipChat integration! With this new connection you can receive notifications from TeamServer to your configured HipChat in a similar format to in-app notifications.
```

Comment:
@jleo32 I would recommend we remove the word "now" and "new". Let's make it so this document has a shelf life longer than a few weeks.